developers/ide/vscode.md

```diff
@@ -15,7 +15,13 @@ The following steps will only need to be done once to setup both VSCode and your
    
 2. Clone the addons (https://github.com/openhab/openhab-addons.git or preferably your own fork) to %BASE%\openhab-addons
    
-3. If you want to setup openHAB code formatting guidelines, add the following to the VSCode settings:
+3. On Windows, VSCode should be configured to use powershell instead of the classic command line. If not configured already, add the following to the VSCode settings:
```

Comment:
```suggestion
1. On Windows, VSCode should be configured to use powershell instead of the classic command line.
If not configured already, add the following to the VSCode settings:
```

One line per sentence and simple numbering of lists. (The build will take car of numbering)